mitosheet/src/components/taskpanes/Graph/GraphSetupTab.tsx

```diff
@@ -26,6 +38,28 @@ const SAFETY_FILTER_ENABLED_MESSAGE = `Turning on Filter to Safe Size only graph
 
 const GRAPHS_THAT_DONT_SUPPORT_COLOR = [GraphType.DENSITY_HEATMAP]
```

Comment:
It's a bit weird that this is a negative thing while the rest of them are positive. But it's cool...